unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
> Then what should be done about this? As far as I can tell, I can remove a decent portion of the lines in this file and building still succeeds. I have no idea of what the implications of that are on the final binaries. So adding an ifdef for a single line seems silly to me when some major cleanup is needed here.

After merging all of the qstrs from all of the places that generate qstrs, duplicates will be removed. So every unique Q(xxx) will be instantiated exactly once in the final binary.

So if FEATURE A needs Q(recv) and FEATURE B needs Q(recv) we should define it for both features. Otherwise, if FEATURE A is enabled and you didn't provide it with FEATURE B, and FEATURE A gets turned off, then FEATURE B will start to fail in the compile.

I'm sure that we're currently not doing this religiously, but that's the philosophy.